public/js/test/sm-test-case/webpack.config.js

```diff
@@ -0,0 +1,8 @@
+
+module.exports = {
+  entry: "./entry.js",
+  output: {
+    filename: "bundle.js"
+  },
+  devtool: "sourcemap"
+}
```

Comment:
Hm, I want it to be a more complicated test case. Also, are those examples things that you want people to run to try out the debugger? I don't think those should be our test cases; the former are things we will want to change over time, but the latter we can't ever change. My test case is never a runnable example, but just data to be consumed by tests.